pkgs/by-name/co/codecrafters-cli/package.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, git
+}:
+
+buildGoModule rec {
+  pname = "codecrafters-cli";
+  version = "34";
+  commit = "38dc748";
+
+  src = fetchFromGitHub {
+    owner = "codecrafters-io";
+    repo = "cli";
+    rev = "v${version}";
+    hash = "sha256-rNWGd+yUDIrL2P+1XEce+HH8KDfThrTPXW+EOrV/n3o=";
+  };
+
+  vendorHash = "sha256-TQcxzfiqKeCQZUKLHnPjBa/0WsYJhER3fmr4cRGFknw=";
+
+  ldflags = [
+    "-s" "-w"
+    "-X github.com/codecrafters-io/cli/internal/utils.Version=${version}"
+    "-X github.com/codecrafters-io/cli/internal/utils.Commit=${commit}"
```

Comment:
```suggestion
    "-X github.com/codecrafters-io/cli/internal/utils.Commit=$(cat ./COMMIT)"
```

Reply:
This doesn't seem to work (I can't get `nix-build` to show me the exact value used, but I think the `$(cat ./COMMIT)` doesn't get evaluated

Reply:
Or we could just remove the commit hash, this would only result in the version being reported as `v34-unknown`